### DISCUSSION TITLE
Screen Flash when adjust the Volume via Monitorcontrol via HDMI

### DISCUSSION THREAD

**Discussion by cyfirst007** [MonitorControl/MonitorControl#1154] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Volume control is not working properly on my new monitor LG Ergo 27UN880 monitor. When I adjust the volume via Monitorcontrol, the monitor screen will flash (turn off and turn on) one second and come back to normal.

### Steps to reproduce

1. adjust volume via Keyboard or scroll the bar in MonitorControl
2.  Screen flash 1 second and Volume adjusted

### Expected behavior

Screen not flash when adjust the volume via MonitorControl

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.5.1
- Mac model: Mac mini 2018
- MonitorControl version: 4.1.0
- Monitor(s): LG Ergo UN880
- Monitor Cable(s)/Connection(s): HDMI 2.0
- Apple Silicon/M1 (yes or no): No
```

**Comment by waydabber** [comment]
You should use USB-C to connect the display. The HDMI port of the 2018 (or later) Minis mess up DDC communication. I think brightness control does not work either (only the bottom 50% which is software based, not DDC).

**Reply by cyfirst007** [reply]
Does Type-C to Displayport cable can work with DDC communication in this case?

**Reply by waydabber** [reply]
Yes, it should work.

**Reply by cyfirst007** [reply]
It works. Thanks a lot.
